packages/tooltip/src/use-tooltip.ts

```diff
@@ -74,11 +74,11 @@ export function useTooltip(props: UseTooltipProps = {}) {
     modifiers,
   } = props
 
-  const { isOpen, onOpen, onClose } = useDisclosure({
+  const { isOpen, onOpen: _onOpen, onClose: _onClose } = useDisclosure({
```

Comment:
Instead of renaming the functions to `_onOpen` and `_onClose`. I'll prefer it if you renamed the prop names.

`onOpen: onOpenProp`
`onClose: onCloseProp`

Reply:
I will make these changes @segunadebayo, Thanks.